src/modules/session/slice.ts

```diff
@@ -8,7 +8,7 @@ interface SessionState {
 }
 
 const initialState: SessionState = {
-  isLoggedIn: false,
+  isLoggedIn: true, // set to true for development and tesing
```

Comment:
I think you forgot to change this back

Reply:
I was thinking that this should stay true until we get persistent authentication. What do you think?

Reply:
Fair point, that probably makes sense